docs/software/mqtt/index.mdx

```diff
@@ -25,22 +25,34 @@ IMPORTANT: When MQTT is turned on, you are potentially broadcasting your entire
 
 ### MQTT Topics
 
-The device will uplink and downlink raw ([protobuf](https://developers.google.com/protocol-buffers)) packets to the `msh/` prefix:
+If no specific [root topic](/docs/settings/moduleconfig/mqtt#root-topic) is configured, the default root topic will be `msh/`.
+Each device that is connected to MQTT will publish its MQTT state ("`online`"/"`offline`") to:
 
-`msh/2/c/ShortFast/!12345678` where
+`msh/2/stat/USERID`, where `USERID` is the user ID of the gateway device (the one connected to MQTT).
 
-- `!12345678` is the address of the gateway device.
-- `ShortFast` is the channel name.
+For each channel where uplink and/or downlink is enabled two other topics might be used.
+
+#### Protobufs topic
```

Comment:
It might be helpful to link early on to the Mestastic protobuf docs ( https://github.com/meshtastic/protobufs/blob/master/meshtastic) and mention that a bunch of this stuff is documented there?

Once directed to the protobuf docs, that feels like the "API documentation" in essence.

Reply:
Also added in https://github.com/meshtastic/meshtastic/pull/746/commits/ee40ecec4ddbaa3a125be2bf1b77fabd720c3c45